client/client.go

```diff
@@ -94,16 +94,24 @@ func ValidRedirectURL(rURL *url.URL, redirectURLs []url.URL) (url.URL, error) {
 	return url.URL{}, ErrorInvalidRedirectURL
 }
 
-func ClientsFromReader(r io.Reader) ([]Client, error) {
+// LoadableClient contains sufficient information for creating a Client and its related entities.
+type LoadableClient struct {
```

Comment:
I guess I'd like both to just be private so we aren't encouraged to use this in other packages. But yeah, this seems consistent with a lot of our other code, so sound good.